### ISSUE TITLE
Define an official grammar for documentation and site in Bevy

### ISSUE THREAD

**Issue by afonsolage** [bevyengine/bevy#11941]
## How can Bevy's documentation be improved?

While reviewing some PRs from Migration Guide and Bevy site in general I saw a lot of fixes which where just converting from "en-GB" to "en-UK". Since Bevy doesn't have stated somewhere (AFAIK) which one is the "official" Bevy one, those can't be considered valid fixes, since both are valid english.

It would be nice if Bevy explicit define somewhere, maybe on `CONTRIBUTING.md`, which gamar to use while writing docs or any formal communication or even if it allows any "en" variation, and such, we don't consider those conversions to be valid fixes anymore.

**Comment by bushrat011899** [comment]
Beyond documentation, it would be good to have an official language for code as well. As an Australian, I'd love to write centre, colour, and stabilise (instead of center, color, and stabilize) as an example. But, I operate under the _assumption_ that an `en-US` style is preferred, despite not seeing anything confirming that assumption.

Perhaps under the `CONTRIBUTING.md` document in `## How you can help` there could be a section such as `### Standards` to put things like preferred languages, grammar, etc.?

**Comment by BD103** [comment]
Yeah, this popped up in https://github.com/bevyengine/bevy-website/pull/1003#discussion_r1493438634 as well. The blog posts used US spelling in the past, but it would be nice to codify it.
